### PR TITLE
fix: update chapter 2 challenge 2 copy in English

### PR DESCRIPTION
**Pull request by pinheadmz**
closes https://github.com/saving-satoshi/saving-satoshi/issues/597

Well, at least the first two bullet points. I can't recall what this refers to:
> Should we also provide the function code inside the terminal like Kevin showed us on the call today?

### CODE REVIEW

File: i18n/locales/en.ts [hunk 1]
```diff
@@ -311,20 +311,18 @@ const translations = {
...
+        'Alright, time to write and run your own code. Write a script that generates a sha256 hash that begins with five zeroes ("00000..."). Your code should repeatedly call the sha256 function with different input until the output satisfies this requirement. You should try incrementing an integer inside a loop to get different inputs. In cryptography this number may referred to as a "nonce" or "number used once".',
       python: {
-        paragraph_three:
-          'We are using the hashlib library in python to help you along in creating this function, you can use the following external resources to help you write this function if needed:',
+        paragraph_two:
+          "When you find a nonce with a hash that begins with five zeroes, use Python's `print()` function to return the hash to the console. We are using the hashlib library in python to help you along in creating this function, you can use the following external resources to help you write this function if needed:",
```
Comment (pinheadmz) [inline i18n/locales/en.ts:317]: hm I had written this with single quotes and an escaped `Python\'s` but i think a git hook linter in the project changed it?

**Comment by ecurrencyhodler** [comment]
Hey Matthew. Thanks for submitting this PR. Let's discuss this on our call tomorrow.

**Review by afterburn — APPROVED** [review]
lgtm

File: i18n/locales/en.ts (hunk 1 above)
Comment (afterburn) [reply i18n/locales/en.ts:317]: @pinheadmz We do have a pre commit hook that runs the `lint-staged` command you can find in `package.json`, which runs prettier to format this doc. But IIRC we don't have any rule that would do this. Could it be your own editor settings? 

In any case this should work: `"When you find a nonce with a hash that begins with five zeroes, use Python's 'print()' function to return the hash to the console. We are using the hashlib library in python to help you along in creating this function, you can use the following external resources to help you write this function if needed:"` as you can see in 7a18e5a160ecb1d4955a86adf1dbdb6681284108

**Comment by benalleng** [comment]
I think it's fine as is but we do have the ability to add styles to text with an inline `<span>` to get some "code text" effects
image

**Pull request merged**